### ISSUE TITLE
Ember Global deprecation warning due to old version of ember-cli-babel in ember-animated

### ISSUE THREAD

**Issue by rahulk94** [ember-animation/ember-animated#367]
We're looking to upgrade our application from Ember 3.24 to 3.28 (latest LTS) and have come across the following deprecation warnings due to `ember-animated`'s (`0.11.0`) use of `ember-cli-babel@6`.
```
WARNING: [DEPRECATION] [DEPRECATION] Usage of the Ember Global is deprecated. You should import the Ember module or the specific API instead.

See https://deprecations.emberjs.com/v3.x/#toc_ember-global for details.

Usages of the Ember Global may be caused by an outdated ember-cli-babel dependency. The following steps may help:

...

The following outdated versions are found in your project:

* ember-cli-babel@6.18.0, currently used by:
  * ember-angle-bracket-invocation-polyfill@2.0.2 (Dormant)
    * Depends on ember-cli-babel@^6.17.0
    * Added by ember-animated@0.11.0
  * ember-maybe-import-regenerator@0.1.6 (Dormant)
    * Depends on ember-cli-babel@^6.0.0-beta.4
    * Added by ember-animated@0.11.0
  * ember-named-arguments-polyfill@1.0.0 (Dormant)
    * Depends on ember-cli-babel@^6.6.0
    * Added by ember-animated@0.11.0
  ```

It looks like this has been fixed on master already so the fix for this would be included the the subsequent release. Are there any timelines around this?

**Comment by ef4** [comment]
Released in 0.12.0.

**Comment by rahulk94** [comment]
Perfect, thanks @ef4
